### DISCUSSION TITLE
Enhancement: Fix the license

### DISCUSSION THREAD

**Discussion by Sudospective** [FunkinCrew/Funkin#1662] [Ideas]
You cannot have a legal document that dictates the use of your program be a single line that doesn't specify anything other than open sourcing mods. This not only leaves YOU vulnerable for liability for literally anything, but leaves the user vulnerable for the same. You need to fix this before someone gets the brilliant idea of "suing you for slow game".

There's also literally nothing stopping me from taking the entire game, changing virtually nothing about it, and selling the entire game for a profit. That's not a mod, that's a fork, and your license has no prevention against this. There's also nothing stopping me from filing a lawsuit claiming I own the original after covering any trace of it being created after the true original. I understand taking the piss out of software licensing, but at least take YOURSELVES seriously.

**Comment by EntropiaFox** [comment]
Chiming in as well to say that while the previous Apache 2.0 license did not require re-licensing under the same license as the original work (e.g. in case of a mod or a fork of the project), something like the [GPLv3](https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3)) would fulfill that requirement while also being a _real_ license not leaving you open to liability.

**Comment by Wilker-uwu** [comment]
i'd like to suggest the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) over the standard GPL, as it also enforces distribution of the source code even if you distribute the game itself on the site as an embedded or something. and because most websites are likely not willing to do that, and you'd like to promote the Newgrounds page the most, i think that would help, maybe.
